### PR TITLE
[FIX] account_amortization.

### PR DESCRIPTION
**Pull request by stundzia**
Fixed issue of doc_type not being passed from supplier invoice, when creating assets or amortization expenses. Fixed poliza english translation (policy). Fixed depreciation spelling. Fixed demo data
assets and categories not being shown (made asset domain include assets with doc_type=False). Fixed help field declaration in view.

### CODE REVIEW

File: account_amortization/model/account_asset_invoice.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                    'currency_id': line.invoice_id.currency_id.id,
+                    'purchase_date' : line.invoice_id.date_invoice,
+                    'doc_type' : line.asset_category_id.doc_type,
+                }
+                changed_vals = asset_obj.onchange_category_id(cr, uid, [], vals['category_id'], context=context)
+                vals.update(changed_vals['value'])
+                asset_id = asset_obj.create(cr, uid, vals, context=context)
+                if line.asset_category_id.open_asset:
+                    asset_obj.validate(cr, uid, [asset_id], context=context)
+        return True
```
Comment (nhomar) [inline account_amortization/model/account_asset_invoice.py]: fix the EOF pliz.

**Comment by nhomar** [comment]
@stundzia Thanks for the contribution. Please check the travis status there are some little LINT errors can you help us to fix them?

Regards.

File: account_amortization/__openerp__.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 ##############################################################################
 {
     "name": "Vauxoo Account Amortization",
-    "version": "0.6",
+    "version": "0.61",
```
Comment (osvalr) [inline account_amortization/__openerp__.py]: Did you mean 0.6.1? if not, why the big jump to 0.61?

Comment (stundzia) [inline account_amortization/__openerp__.py]: Yes, my bad.

**Pull request merged**